js/form.js

```diff
@@ -141,10 +150,39 @@ priceForm.addEventListener('input', () => {
   priceForm.reportValidity();
 });
 
+const setMainFormSubmit = () => {
```

Comment:
```suggestion
const handleFormSubmit = () => {
```
handle - обработать что-то
Form - у нас одна форма, зачем Main ей еще писать?